protocol/rpcconsumer/rpcconsumer_server.go

```diff
@@ -154,11 +154,14 @@ func (rpccs *RPCConsumerServer) waitForPairing() error {
 		}
 	}()
 
+	numberOfTimesChecked := 0
 	select {
 	case <-reinitializedChan:
 		break
 	case <-time.After(30 * time.Second):
-		return utils.LavaFormatError("failed initial relays, csm was not initialized after timeout", nil,
+		numberOfTimesChecked += 1
```

Comment:
numberOfTimesChecked is always 1, this bails the select and returns nil, was a for loop of retries intended?